IotTemplates/WindowsIotCoreTemplates/ProjectTemplates/WiringAppCpp/Package-native.appxmanifest

```diff
@@ -54,5 +54,10 @@
     <Capability Name="internetClient" />
     <iot:Capability Name="lowLevelDevices" />
     <DeviceCapability Name="109b86ad-f53d-4b76-aa5f-821e2ddf2141"/>
+    <DeviceCapability Name="serialcommunication">
```

Comment:
why is a new capability needed?  i don't see new code ... does lightning require serialcommunication?